### PR TITLE
OBPIH-6153 Fix assigning bidirectional association between Picklist a…

### PR DESCRIPTION
**Pull request by kchelstowski**
…nd Requisition

TL:DR - we've been living with this bug forever, but it has not been yet noticed, because we have not returned the `picklist` in the `toJson` of the `StockMovement`, that I've added recently.
The problem is that the 1:1 association between requisition and picklist has been missing one part - we set the requisition for the picklist, but we don't set the picklist for the requisition - that caused an issue that when serializing the stock movement (in toJson), the requisition's picklist is in fact `null`.
The missing association is obviously detected and added automatically by Hibernate, but after the session is closed, so after refreshing the page and re-fetching the SM, everything works correctly, because the association is added, but it didn't work (picklist was treated as null) when returning the SM from the initial request. 

If anyone is interested in details of that issue, let me know, so I can explain it in more details or record a video with the debugger, but I hope it is kinda self-explanatory.

### CODE REVIEW

**Review by EWaterman — APPROVED** [review]

**Review by drodzewicz — APPROVED** [review]

**Review by jmiranda — CHANGES_REQUESTED** [review]
Don't merge this. Let's discuss on the tech huddle tomorrow.

**Review by jmiranda — APPROVED** [review]
